backend/contentBackend.js

```diff
@@ -243,8 +241,9 @@ async function getContentMetadata() {
  * @return {object} - metadata of content
  */
 /** @expose */
-export async function getSiteMetadata(jwt) {
-  await validateJWT(jwt);
+export async function getSiteMetadata() {
```

Comment:
Here too, please fix JavaDoc.

Generally IIUC `git grep -i jwt; git grep -i token -- src/ backend/` should return _no hits_ on this branch.